### DISCUSSION TITLE
An unsolicited positive appraisal -- may be used for marketing

### DISCUSSION THREAD

**Discussion by csrabak** [fesch/Structorizer.Desktop#1107] [General]
I'm part of a small team working with embedded systems for a lot of time, and we know NS diagramming tools since the era a tool, nowadays still marketed by a German firm, was distributed by Siemens AG.

My colleagues appraised well the feature Structorizer has of "correctly" joining cases in C language when they are not separated by a `break`, a common idiom used in this programming language.

According to the not asked for report, the commercial offerings, an American in addition to the European one, aren't able to do so, leaving an empty case in the extracted diagrams of C sources.

I don't know how to put a label in the initial post so I'll leave this to the admins.

HTH

**Comment by codemanyak** [comment]
We feel flattered 🥲, thanks. Well, we try our best to make as much sense as possible of the features of Structorizer.

I provisionally invented a new label for this kind of remark, though it is somehow vaguely reminiscent of some (hallucinated?) GitHub feature where discussions about a project can be run. It wasn't the wiki, I guess, but I don't get back to it just now.

**Comment by codemanyak** [comment]
I wrote:
> it is somehow vaguely reminiscent of some (hallucinated?) GitHub feature where discussions about a project can be run. It wasn't the wiki, I guess, but I don't get back to it just now.

Now I found it again: It's [GitHub Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). Only the owner of a repository can enable it. (So it's up to Bob's consent.) If enabled, it will occur among the top menu entries of the repository. That does not of course mean that it will actively be used, and I'm not quite sure who can see and use it, i.e., how the "community" of the repository manifests. Will have to read in detail the documentation...

**Comment by codemanyak** [comment]
As for the "marketing", I referenced your praise on the [news panel of the Structorizer home page](https://structorizer.fisch.lu/index.php?include=news).
But actually, this is what a correct code import is expected to do, and it was not rocket science to implement it.

**Comment by fesch** [comment]
Ohh .... I didn't know about the "Discussion" feature, but found quickly the corresponding setting on how to enable it. Let's see how it works and what it brings ;-)

**Comment by csrabak** [comment]
I hope the Discussion thing foments interactions which could enhance even further Structorizer
